mobile/src/defaultViewComponents/defaultViewTableCell.js

```diff
@@ -82,13 +81,13 @@ export default class DefaultViewTableCell extends Component {
   renderCellButtons = () => {
     return (
       <View style={s.buttonBox}>
-        <TouchableOpacity onPress={() => client.openURL(`dd://profile/${this.props.item.creator.id}`)} style={s.largeButton}>
+        { this.props.item.creator.id !== client.currentUser.id ? <TouchableOpacity onPress={() => client.openURL(`dd://profile/${this.props.item.creator.id}`)} style={s.largeButton}>
```

Comment:
React will also not render a `false` value, so
```jsx
{ x !== y && <TouchableOpacity ... /> }
```
works as well without the ternary operator